src/deadline/cinema4d_submitter/scene.py

```diff
@@ -170,6 +174,43 @@ def get_render_data(doc=None, take=None):
             render_data = doc.GetActiveRenderData()
         return render_data
 
+    @staticmethod
+    def get_output_paths(take=None) -> str:
+        doc = c4d.documents.GetActiveDocument()
+        doc_path = doc.GetDocumentPath()
+        if not take:
+            take_data = doc.GetTakeData()
+            take = take_data.GetCurrentTake()
+        render_data = Scene.get_render_data(doc=doc, take=take)
+        rbc = render_data.GetDataInstance()
```

Comment:
Nit: Can we have better variables names? It is not intuitive as what rbc or rpd mean. :)

Reply:
I agree names could be better but they match the dict key names in the c4d api here which also uses [_rBc](https://developers.maxon.net/docs/py/2024_0_0a/modules/c4d.modules/tokensystem/index.html#:~:text=_rData%3A%20RenderData-,_rBc,-%3A%20BaseContainer) - I could rename it `render_base_container_instance` ?

Reply:
That would make more sense, thanks!